eks-hybrid-iam-ra-setup.sh

```diff
@@ -51,6 +67,15 @@ cat << EOF > "${SECRETS_DIR}/node.key"
 ${NODE_KEY_DATA}
 EOF
 
+if [ "${ENABLE_CREDENTIALS_FILE}" = "true" ]; then
+    export EKS_HYBRID_SHARED_CREDENTIALS_FILE="/var/eks-hybrid/.aws/credentials"
+    export SIGNING_HELPER_UPDATE_COMMAND=$(get_aws_signing_helper_update_command ${SECRETS_DIR}/config)
```

Comment:
```suggestion
    EKS_HYBRID_SHARED_CREDENTIALS_FILE="/var/eks-hybrid/.aws/credentials"
    SIGNING_HELPER_UPDATE_COMMAND=$(get_aws_signing_helper_update_command "${SECRETS_DIR}/config")
    export EKS_HYBRID_SHARED_CREDENTIALS_FILE SIGNING_HELPER_UPDATE_COMMAND
```
To avoid the return code being dropped, I would at least export SIGNING_HELPER_UPDATE_COMMAND after it's been assigned.

Reply:
Done